client/webserver/api.go

```diff
@@ -278,7 +280,7 @@ func (s *WebServer) apiCancel(w http.ResponseWriter, r *http.Request) {
 	}
 	err := s.core.Cancel(form.Pass, form.OrderID)
 	if err != nil {
-		s.writeAPIError(w, core.NewError("error cancelling order %s: %v", form.OrderID, err))
+		s.writeAPIError(w, errors.New(fmt.Sprintf("error cancelling order %s: %v", form.OrderID, err)))
```

Comment:
These `errors.New(fmt.Sprintf` should be replaced with `fmt.Errorf`.

Also, when there is an `err` as one of the printf args, the specifier should be `%w` so that errors.Is/As can unwrap the core.Error (or whatever type it is)

Reply:
Got it, my bad.